scripts/server.js

```diff
@@ -39,7 +39,9 @@ app.listen(app.get('port'), () => {
       }).`,
       app.get('port'),
     );
-    console.log('\n  Press CTRL-C to stop');
+    console.log('');
+    console.log('  Press CTRL-a-k to stop');
+    console.log('  Press CTRL-a-? for help');
```

Comment:
That must be... me, but I cannot exit `stmux` easily 😞 

Note that it should still be `CTRL-C` to stop the server, because that's only one process in one stmux panel. I'd suggest adding a new sentence for exiting `stmux` (and keep CTRL-C).

Reply:
Ah ok, got it. So, `stmux` uses `CMD` and not `CTRL` on MacOS. `CMD-a-k` exits properly now, no need for `killall` anymore 😅

Reply:
I cannot get any of these commands to work for me to exit everything. I'm on OS X. What is the exact keystroke combination I should be using?

Reply:
While maintaining `CMD`, I hit `a` then `k`

Reply:
Huh. For me in iTerm on macOS, control works as expected. I press and hold control, press a, let go of control, and press k.

Reply:
Heh. I should have written "I can easily imagine issues with various **Mac**OS" in my review comment 😅